docs/tutorial/troubleshooting/migrate_pool_by_migrating_disks.md

```diff
@@ -0,0 +1,314 @@
+# Pool Migration when underlying disks were migrated to different node
```

Comment:
This looks elevated IMO.
This heading could be #Node Replacement simply.

Reply:
Usually when the term `Node Replacement`  heard they may think the case where enter stack is replaced i.e node and underlying pool disks also.